behandlingslager/domene/src/main/java/no/nav/foreldrepenger/behandlingslager/behandling/ytelsefordeling/YtelsesFordelingRepository.java

```diff
@@ -59,7 +59,8 @@ private YtelseFordelingAggregat mapEntitetTilAggregat(YtelseFordelingGrunnlagEnt
             .medSakskompleksDekningsgrad(ytelseFordelingGrunnlagEntitet.getSakskompleksDekningsgrad() == null ? null : Dekningsgrad.grad(
                 ytelseFordelingGrunnlagEntitet.getSakskompleksDekningsgrad()))
             .medOppgittRettighet(ytelseFordelingGrunnlagEntitet.getOppgittRettighet())
-            .medOverstyrtRettighet(ytelseFordelingGrunnlagEntitet.getOverstyrtRettighet())
+            .medAvklartRettighet(ytelseFordelingGrunnlagEntitet.getOverstyrtRettighet())
```

Comment:
Her er det litt forvirrende å sette avklartRettighet og bruke get metoden for getOverstyrtRettighet(). Rename til getAvklartRettighet?